hardware/areca.py

```diff
@@ -147,6 +150,13 @@ def _disable_password():
 
 def detect():
     """Detect Areca controller configuration."""
+    if not detect_utils.which('cli64'):
+        sys.stderr.write('Cannot find cli64 binary\n')
+        return []
```

Comment:
I think we never reached the return None before, while in this case we care to return an empty list because in the cli script we're extending the list with this one

Reply:
Then probably return [] in the end of detect_areca